.cci.jenkinsfile

```diff
@@ -52,14 +52,18 @@ pod(image: 'registry.fedoraproject.org/fedora:32', runAsUser: 0, kvm: true, memo
 
       stage("Image tests") {
         try {
-          shwrap("cd /srv && rm tmp/kola -rf")
-          shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S")
-          shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --scenarios iso-offline-install")
-          // and again this time with the 4k image
-          shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --qemu-native-4k --no-pxe")
+          parallel default_scenarios: {
```

Comment:
Won't we potentially hit RAM issues here? I think we require 4G per scenario.

Reply:
Yes, you're right. I briefly thought about this, but when I saw we were requesting 9G, I sort of assumed that should be more than enough...

Anyway, I changed tactics here so that we test `iso-offline-install` as a default scenario. I also moved out `iso-install` since I think osmet is going to be the hot path now that it's supported everywhere, though I can put it back if wanted.